calico/AUTHORS.md

```diff
@@ -203,18 +202,11 @@ by GitHub for the core repositories within the projectcalico/ organization. It i
 | Leo Ochoa                                | @leo8a |
 | libby kent                               | @libby |
```

Comment:
Hmm, some of these name removes are suspect. Guessing there is something wrong with our script :cry: